chall07/pcuadrad.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"strconv"
+	"log"
+	"os"
+	"time"
+)
+
+func RGBToHex(r int, g int, b int) (string) {
+	return fmt.Sprintf("%x%x%x",r ,g ,b)
```

Comment:
Unfortunately, the RGB format asks for a 2 character precision. Which means that `9` should be writtent `09`. To give you an example, with this input `r=189,g=208,b=9` you should output `bdd009`, but the output you send is `bdd09`.  
To fix this it is quite simple, you just have to modify the printf's precision:
```suggestion
	return fmt.Sprintf("%02x%02x%02x",r ,g ,b)
```

Reply:
It is true! I didn't pay attention :(